<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fichePartenariat</class>
 <widget class="QDialog" name="fichePartenariat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_partenaire">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>28</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_duree">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>91</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_partenaire">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>34</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Partenaire </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dateDebut">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>65</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date debut </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_duree">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Duree</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_type">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>121</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_anneeReconduction">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>168</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Annee reconduction </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_anneeReconduction">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>169</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_description">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Description </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_commentaire">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Commentaire </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_poc">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>198</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>POC  </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_poc">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>199</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_realisations">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Realisations </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pays">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>229</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pays">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>228</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pays</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_dateDebut">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>163</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_supprimerPartenariat">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sauvegarderPartenariat">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_bloquedPartenariat">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Debloquer</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_commentaire">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_description">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_partenaireControl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>29</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_dateDebutControl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_dureeControl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>91</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_typeControl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>123</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_anneeReconductionControl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pocControl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_realisationsControl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>283</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_paysControl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_descriptionControl">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>343</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_commentaireControl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>404</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_commentaire">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>483</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_realisations">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
